wasm/read.go

```diff
@@ -35,6 +35,17 @@ func readBytes(r io.Reader, n int) ([]byte, error) {
 	return nil, io.ErrUnexpectedEOF
 }
 
+func WriteByte(w io.Writer, b byte) error {
+	_, err := w.Write([]byte{b})
+	return err
+}
+
+func ReadByte(r io.Reader) (byte, error) {
```

Comment:
perhaps not export it?
```suggestion
func readByte(r io.Reader) (byte, error) {
```

Reply:
The `ReadByte` method is used in `disasm` package.

Reply:
ah. right.